tests/pytorch/nightly/common.libsonnet

```diff
@@ -137,6 +137,39 @@ local volumes = import 'templates/volumes.libsonnet';
     },
   },
 
+  Accelerate:: {
+    local config = self,
+    tpuSettings+: {
+      tpuVmExports+: |||
+        export PATH=~/.local/bin:$PATH
+      |||,
+      tpuVmExtraSetup: |||
+        git clone https://github.com/huggingface/accelerate.git
+        pip install -e accelerate
```

Comment:
can we move this part to common? I think a couple HF test failed because they also need accelerate